src/motion_detector/motion_detector.py

```diff
@@ -194,6 +197,50 @@ async def get_object_point_clouds(self,
                                       **kwargs) -> List[PointCloudObject]:
         raise NotImplementedError
     
+    async def get_properties(
+            self,
+            *,
+            extra: Optional[Mapping[str, Any]] = None,
+            timeout: Optional[float] = None) -> Vision.Properties:
+                return Vision.Properties(
+                classifications_supported=True,
+                detections_supported=True,
+                object_point_clouds_supported=False,
+        )
+        
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+        result = CaptureAllResult()
+        if camera_name != self.cam_name:
+            raise Exception(
+                "Camera name passed to method:",camera_name, "is not the configured 'cam_name':", self.cam_name)
+        self.camera.get_image()
+        img = await self.camera.get_image(mime_type="image/jpeg")
+        if return_image:
+            result.image = img
+        if return_classifications:
+            classifs = await self.get_classifications(img, 1)
+            result.classifications = classifs
+        if return_detections:
+            dets = await self.get_detections(img)
+            result.detections = dets 
+        # No object point clouds
```

Comment:
Assuming I shouldn't even log

Reply:
correct. It will just always return empty